daisy/workflow/workflow.go

```diff
@@ -245,6 +257,49 @@ func (w *Workflow) populateStep(step *Step) error {
 	return step.SubWorkflow.workflow.populate()
 }
 
+func (w *Workflow) populateMergeWorkflow(step *Step) error {
+	step.MergeWorkflow.workflow.GCSPath = w.GCSPath
+	step.MergeWorkflow.workflow.Name = step.name
+	step.MergeWorkflow.workflow.Project = w.Project
+	step.MergeWorkflow.workflow.Zone = w.Zone
+
+	for k, v := range step.MergeWorkflow.Vars {
+		step.MergeWorkflow.workflow.AddVar(k, v)
+	}
+	if err := step.MergeWorkflow.workflow.populateVars(); err != nil {
+		return err
+	}
+
+	var replacements []string
+	for k, v := range w.autovars {
+		replacements = append(replacements, fmt.Sprintf("${%s}", k), v)
+	}
+	for k, v := range step.MergeWorkflow.workflow.vars {
+		replacements = append(replacements, fmt.Sprintf("${%s}", k), v.Value)
+	}
+	substitute(reflect.ValueOf(step.MergeWorkflow.workflow).Elem(), strings.NewReplacer(replacements...))
+
+	for name, s := range step.MergeWorkflow.workflow.Steps {
```

Comment:
No, populates is slightly different, this one specifically overwrites the steps w field (so they share like 2 actual lines?)